.travis.yml

```diff
@@ -40,8 +53,9 @@ install:
   - export COMPOSER_EXIT_ON_PATCH_FAILURE=1
 
   # MySQL Options
-  - mysql -e 'SET GLOBAL wait_timeout = 5400;'
-  - mysql -e "SHOW VARIABLES LIKE 'wait_timeout'"
+  - mysql -u root -e 'CREATE DATABASE drupal;'
+  - mysql -u root -e 'SET GLOBAL wait_timeout = 5400;'
+  - mysql -u root -e "SHOW VARIABLES LIKE 'wait_timeout'"
```

Comment:
yeah - that's almost definitely left over from whatever this was based on originally. yank it.